worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/BukkitAdapter.java

```diff
@@ -360,7 +367,13 @@ public static BlockType asBlockType(Material material) {
         if (!material.isBlock()) {
```

Comment:
might be able to skip this (and the `isItem` below) as the EnumMap stores a `null` object for us and can just skip `Material`'s switches.

Reply:
The javadoc for Map's computeIfAbsent says `If the specified key is not already associated with a value (or is mapped to null)`, so this may cause it to try to re-compute every time